WooCommerce/Classes/ViewRelated/Blaze/BudgetSetting/BlazeBudgetSettingView.swift

```diff
@@ -0,0 +1,301 @@
+                    .bold()
+                    .largeTitleStyle()
+
+                Text(Localization.totalSpend)
+                    .subheadlineStyle()
+            }
+
+            // Daily amount slider and estimated impression
+            VStack {
+                Text(viewModel.dailyAmountText)
+
+                Slider(value: $viewModel.dailyAmount,
+                       in: viewModel.dailyAmountSliderRange,
+                       step: BlazeBudgetSettingViewModel.Constants.dailyAmountSliderStep)
+
+                AdaptiveStack {
+                    Text(Localization.estimatedImpressions)
+                    Image(systemName: "info.circle")
+                }
+                .font(.subheadline)
+                .onTapGesture {
+                    showingImpressionInfo = true
+                }
+
+                // TODO: fetch impressions and display
+                Text("2,588 - 3,458")
+                    .bold()
+                    .font(.subheadline)
+            }
+        }
+    }
+
+    var footerView: some View {
+        VStack(alignment: .leading, spacing: Layout.sectionContentSpacing) {
+            Divider()
+
+            // Duration title
+            Text(Localization.duration)
+                .secondaryBodyStyle()
+                .padding(.horizontal, Layout.contentPadding)
+                .padding(.top, Layout.sectionContentSpacing)
+
+            // Formatted duration
+            HStack {
+                Text(viewModel.formattedDateRange)
+                    .bold()
+                    .bodyStyle()
+                Text("·")
+                Text(Localization.edit)
+                    .foregroundColor(.accentColor)
+                    .bodyStyle()
+            }
+            .padding(.horizontal, Layout.contentPadding)
+            .onTapGesture {
+                showingDurationSetting = true
+            }
+
+            // CTA to confirm all settings
+            Button(Localization.update) {
+                viewModel.confirmSettings()
+                dismiss()
+            }
+            .buttonStyle(PrimaryButtonStyle())
+            .padding([.horizontal, .bottom], Layout.contentPadding)
+            .padding(.top, Layout.sectionContentSpacing)
+        }
+        .background(Color(.systemBackground))
+    }
+
+    var impressionInfoView: some View {
+        NavigationView {
+            ScrollView {
+                Text(Localization.impressionInfo)
+                    .bodyStyle()
+                    .padding(Layout.contentPadding)
+            }
+            .navigationTitle(Localization.title)
+            .navigationBarTitleDisplayMode(.inline)
+            .toolbar {
+                ToolbarItem(placement: .confirmationAction) {
+                    Button(Localization.done) {
+                        showingImpressionInfo = false
+                    }
+                }
+            }
+        }
+        .padding(.top)
+    }
+
+    var durationSettingView: some View {
```

Comment:
In landscape mode I can't seem to dismiss this view:

https://github.com/woocommerce/woocommerce-ios/assets/266376/4ce46518-f830-4f22-9ede-4053252ef0ea

Reply:
This can still be dismissed by tapping Apply. But good point about no Dismiss button - I consulted Ann on Figma and will update after getting a confirmation from her.

Reply:
In portrait mode I can drag down the sheet to make it disappear. That's the same action I wanted to do here in landscape mode, but the problem seems like the dragging bar at the top is not shown in landscape 🤔

Reply:
This is the behavior for iOS - the partial modal is only available if the vertical size class is not compact. Ann confirmed on Figma to add a button to dismiss this view, so I updated that in 5478da960984de3a6f5a3a31ef111aba74e2fe19. To simplify the alignment of the title, I use a navigation bar so there's a divider below the bar as the default behavior.